src/SonarScanner.MSBuild.Common/ProjectInfo/ProjectInfoValidity.cs

```diff
@@ -30,5 +30,6 @@ public enum ProjectInfoValidity
     InvalidGuid,
     ExcludeFlagSet,
     NoFilesToAnalyze,
-    DuplicateGuid
+    DuplicateGuid,
+    NoProjectFound,
```

Comment:
Can we rename this to `NotFound`?

Reply:
I renamed it to `ProjectNotFound`, to be a bit more specific